### PR TITLE
[DOC] Add LDAP document

### PR DESCRIPTION
**Pull request by zhaohehuhu**
### _Why are the changes needed?_

I'd like to update LDAP doc to guide users for setup LDAP authentication in Kyuubi.
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

image

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/security/ldap.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Enable LDAP Authentication
+
+To enable the LDAP authentication, we need to
+
+- Configure the following authentication properties to `$KYUUBI_HOME/conf/kyuubi-defaults.conf`
+  on each node where kyuubi server is installed.
+
+## Configure the authentication properties
+
```
Comment (pan3793) [inline docs/security/ldap.md:31]:
```suggestion
- Configure the following authentication properties to `$KYUUBI_HOME/conf/kyuubi-defaults.conf`
  on each node where kyuubi server is installed.

## Configure the authentication properties

```

File: docs/security/ldap.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Configure the following properties to `$KYUUBI_HOME/conf/kyuubi-defaults.conf` on each node where kyuubi server is
+installed.
+
+```properties example
+kyuubi.authentication=LDAP
+kyuubi.authentication.ldap.baseDN=dc=com
+kyuubi.authentication.ldap.domain=kyuubi.com
+kyuubi.authentication.ldap.binddn=uid=kyuubi,OU=Users,DC=apache,DC=org
+kyuubi.authentication.ldap.bindpw==kyuubi123123
```
Comment (pan3793) [inline docs/security/ldap.md:35]:
```suggestion
kyuubi.authentication.ldap.bindpw=kyuubi123123
```

File: docs/security/ldap.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+# Configure Kyuubi to Use LDAP Authentication
+
+Kyuubi supports authentication via LDAP. Users can log into Kyuubi using their credentials. Then, Kyuubi can request
+LDAP server to verify user's identity and grant right access permissions based on their LDAP roles and groups.
+
+## Enable LDAP Authentication
+
+To enable the LDAP authentication, we need to
+
+Configure the following properties to `$KYUUBI_HOME/conf/kyuubi-defaults.conf` on each node where kyuubi server is
```
Comment (pan3793) [inline docs/security/ldap.md:27]:
```suggestion
To enable the LDAP authentication, we need to configure the following properties to
`$KYUUBI_HOME/conf/kyuubi-defaults.conf` on each node where kyuubi server is
```

File: docs/security/ldap.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+# Configure Kyuubi to Use LDAP Authentication
+
+Kyuubi supports authentication via LDAP. Users can log into Kyuubi using their credentials. Then, Kyuubi can request
+LDAP server to verify user's identity and grant right access permissions based on their LDAP roles and groups.
+
+## Enable LDAP Authentication
+
+To enable the LDAP authentication, we need to configure the following properties to
+`$KYUUBI_HOME/conf/kyuubi-defaults.conf` on each node where kyuubi server is
+installed.
```
Comment (pan3793) [inline docs/security/ldap.md:27]:
```suggestion
To enable the LDAP authentication, we need to configure the following properties to `$KYUUBI_HOME/conf/kyuubi-defaults.conf`
on each node where kyuubi server is installed.
```

**Comment by pan3793** [comment]
Thanks, I think it's good for most LDAP users, but we also support some complex cases like Hive does. We can mention it in short words, and guide users to read the [Hive Wiki](https://cwiki.apache.org/confluence/display/Hive/User+and+Group+Filter+Support+with+LDAP+Atn+Provider+in+HiveServer2) and search the related configurations in Kyuubi configuration page

**Comment by zhaohehuhu** [comment]
> 

Got it. Thanks.

**Comment by zhaohehuhu** [comment]
> Thanks, I think it's clear enough for most LDAP users, but we also support some complex cases like Hive does. We can mention that in short words, and guide users to read the [Hive Wiki](https://cwiki.apache.org/confluence/display/Hive/User+and+Group+Filter+Support+with+LDAP+Atn+Provider+in+HiveServer2) and search the related configurations in Kyuubi configuration page

added. plz review it. Thanks.

**Comment by zhaohehuhu** [comment]
@pan3793 plz help review

**Comment by pan3793** [comment]
Thanks, merged to master/1.7

**Pull request closed without merging**